apps/ai-agent/app/providers/gemini.py

```
@@ -0,0 +1,17 @@
+import json, os, urllib.request
+from .base import LLMProvider
+
+class GeminiProvider(LLMProvider):
+    def __init__(self):
+        self.key = os.getenv('LLM_API_KEY', '')
+        self.model = os.getenv('LLM_MODEL', 'gemini-3.1-flash')
+        self.base = os.getenv('LLM_BASE_URL', 'https://generativelanguage.googleapis.com/v1beta')
+    def decide(self, prompt):
+        if not self.key: raise RuntimeError('LLM_API_KEY is not configured')
```

Ruff (0.16.3)

[error] Multiple statements on one line (colon)

(E701)